src/DataProtection/DataProtection/src/XmlEncryption/EncryptedXmlDecryptor.cs

```diff
@@ -44,6 +45,11 @@ public EncryptedXmlDecryptor(IServiceProvider? services)
     /// </summary>
     /// <param name="encryptedElement">An encrypted XML element.</param>
     /// <returns>The decrypted form of <paramref name="encryptedElement"/>.</returns>
+#pragma warning disable SYSLIB0022 // Rijndael types are obsolete
+    [DynamicDependency(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor, typeof(RijndaelManaged))]
```

Comment:
Is this actually used or does it just get instantiated and ignored?

Reply:
Per our offline discussion, it's used and important.  I might say something like "Rijndael (aka AES) is used in nearly all scenarios.  If we find another important family, we should add it here as well.  In the meantime, a useful exception will be thrown."

Reply:
Added a comment.